web/policies/stripe_customer_policy.ex

```diff
@@ -0,0 +1,12 @@
+defmodule CodeCorps.StripeCustomerPolicy do
+  alias CodeCorps.StripeCustomer
+  alias CodeCorps.User
+  alias Ecto.Changeset
+
+  def create?(%User{id: current_user_id}, %Changeset{changes: %{user_id: user_id}}), do: current_user_id == user_id
+  def create?(%User{}, %Changeset{}), do: false
```

Comment:
Considering how our `ability.ex` is written, it can never be anything other than a `%User{}` and a `%Changeset{}`.